pages/activities.jsx

```diff
@@ -277,7 +277,7 @@ var ActivitiesPage = React.createClass({
               <HiveLink
                 src1x="/img/pages/activities/svg/logo-hive-community.svg"
```

Comment:
Ummm I kinda prefer including suffix for both cases to make them explicit.

I guess for `.svg` cases it makes more sense to have `src` instead of `src1x` since `SVGs` won't have scaling problem.  But for other image types like `JPGs` or `PNGs` I'm worried that people might forget about the `src2x` prop and pass a "@2x" version of image to `src` and use CSS or the `width` or `height` attribute to scale it down?  At least I feel like that's something I might accidentally do... :no_mouth: